### PR TITLE
feat(FN-3052): add screenreader labels to premium payments table checkboxes

### PR DESCRIPTION
**Pull request by RLavender98**
## Resolution :heavy_check_mark:
- Added aria-labels to the premium payments table checkboxes

### CODE REVIEW

File: trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk [hunk 1]
```diff
@@ -38,9 +38,9 @@
         {{ tableHeader({ headerText: 'Total payments received', isNumericColumn: true, enableSorting: enablePaymentsReceivedSorting, ariaSort: 'none' }) }}
         {{ tableHeader({ headerText: 'Status', isNumericColumn: false, enableSorting: true, ariaSort: 'none' }) }}
         {% if userCanEdit %}
-          <th scope="col" class="govuk-table__header">
+          <td scope="col" class="govuk-table__header">
```
Comment (RLavender98) [inline trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk:41]: If this is a `th` the select all checkbox aria-label gets reader out for all the items in the column, but 'Select all' describes the specific checkbox not each item in the column

File: trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts [hunk 2]
```diff
@@ -57,6 +57,11 @@ const getCheckboxIdForFeeRecordsAndStatus = (feeRecords: FeeRecord[], status: Fe
   return `feeRecordIds-${feeRecordIdList}-reportedPaymentsCurrency-${reportedPaymentsCurrency}-status-${status}`;
 };
 
+const getCheckboxAriaLabel = (feeRecords: FeeRecord[]): string => {
+  const feeRecordFacilityIdList = feeRecords.map(({ facilityId }) => facilityId).join(' ');
```
Comment (sw-mattw) [inline trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts:61]: Not sure on this but if we're joining facility ID's like this, is easily distinguishable in the screen reader audio output that they are separate numbers? As in if we have facility ID's 123 and 456, is it clear that these are not a single facility ID?

Comment (RLavender98) [reply trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts:61]: It pauses between the two as if they are separate words, when I added columns it started reading them as 'one million eight hundred and thirty thousand.....' and so on which was I guess also ok, except it was inconsistent between ones where there is only one id which it decided to read as 'one eight three zero' so for consistency removed the commas.

Comment (sw-mattw) [reply trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts:61]: Cool thanks for clarifying

**Review by sw-mattw — APPROVED** [review]

**Pull request merged**